src/main/java/com/google/sps/servlets/RepresentativeListServlet.java

```diff
@@ -0,0 +1,79 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.Constants;
+import io.github.cdimascio.dotenv.Dotenv;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Scanner;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.util.EntityUtils;
+import org.apache.http.HttpEntity;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/* 
+The RepListServlet class takes in an address from the user and call the Google Civic
+Info API to pull the list of representatives relevant for that zipcode and returns
+a json formatted objects which contains corresponding offices and officials
+*/
+
+@WebServlet ("/rep_list")
+public class RepresentativeListServlet extends HttpServlet{
+    Dotenv dotenv = Dotenv.load();
```

Comment:
sorry I missed this earlier. the Dotenv.load() should be in the constructor. Generally, we initialize all instance variables in the constructor. Only static variables are initialized outside the constructor.
Also the Dotenv variable should be private.

In fact, you can remove the dotenv variable and do.

Dotenv.load().get("CIVIC_API_KEY); 

in the constructor.

Also, you can make "CIVIC_API_KEY" a constant in Constants.java.